platform/view/services/view/stream.go

```diff
@@ -0,0 +1,26 @@
+/*
+Copyright IBM Corp. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package view
+
+import (
+	"reflect"
+
+	"github.com/hyperledger-labs/fabric-smart-client/platform/view"
+)
+
+type Stream interface {
+	Recv(m interface{}) error
+	Send(m interface{}) error
+}
```

Comment:
I suggest that we use bytes already here and let the caller perform the serialization.